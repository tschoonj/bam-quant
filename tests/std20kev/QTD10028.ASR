File: C:\xrfdaten\2014KW30\chris\std20kev\std10028.spe
Start: 21.07.2014 18:11:09
Stop: 21.07.2014 18:12:49
BESSY: 298.8045514 298.8005051
IONI: 3.493774E-11 3.379306E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 78033
Run: 100000
Dead: 22
ICR: 72300
OCR: 0
Accumulated BESSY: 59704.9140341
Accumulated Ioni: 6.85417296E-09
N Accumulation: 200
Normfactor:  3.7393544e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
exp_x 91.2603988647461 91.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  28   1      7.472   4884497.      2210.     159.04
