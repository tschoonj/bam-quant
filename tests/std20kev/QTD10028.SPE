$DATA:
0 2000
18445
11926
5160
1599
308
49
14
9
4
3
5
4
1
1
1
4
156
1106
1463
1473
1386
1272
1247
1158
1200
1147
1154
1060
998
986
1007
980
965
954
900
849
868
862
871
795
809
832
791
807
768
793
747
739
755
709
717
685
705
713
691
744
634
637
739
671
662
656
652
659
639
658
628
656
646
633
635
600
591
579
611
611
561
587
569
583
560
586
576
581
549
556
571
564
582
542
593
566
584
506
596
538
573
548
549
609
603
661
657
562
583
590
570
554
546
543
530
566
528
523
498
512
501
568
535
482
459
534
514
508
488
516
523
517
524
550
593
523
443
544
501
468
461
536
541
494
504
527
555
567
602
701
675
641
635
541
524
552
453
522
516
482
546
553
502
490
504
491
481
471
476
512
494
454
512
504
488
463
470
478
475
478
489
452
464
513
460
471
518
477
461
511
474
453
439
453
488
459
473
485
500
496
475
481
493
526
516
491
481
479
468
483
483
495
491
512
485
518
476
501
517
471
489
519
496
534
516
512
462
508
483
504
537
514
520
553
525
449
560
527
527
523
532
497
523
486
511
542
537
510
522
557
548
544
519
529
511
596
567
586
541
589
559
572
581
571
567
577
614
627
580
650
594
651
638
648
636
659
680
655
647
723
744
841
1057
1278
1460
1747
2019
2178
2177
1918
1731
1431
1201
946
863
781
707
728
767
727
774
743
788
753
779
798
811
777
792
822
837
897
863
864
875
957
1027
958
1104
1120
1093
1225
1225
1218
1187
1177
1177
1220
1219
1192
1090
1120
1120
1078
1017
1043
1084
1156
1081
1161
1153
1251
1257
1305
1339
1321
1373
1555
1615
1686
1728
1822
1914
1989
2084
2355
2508
2588
2867
3197
3551
4110
5053
6962
11037
19712
36759
67714
118415
194436
291002
398244
500196
568976
592465
559802
481698
378589
270183
175776
103990
56719
29208
14036
6860
3602
2356
1817
1604
1485
1401
1430
1362
1371
1397
1558
1531
1612
1889
1950
2251
2595
3110
4181
5631
8608
13796
21235
32084
45964
60974
75598
86735
90708
86785
76097
61589
45445
31270
19702
11399
6089
3126
1493
683
368
155
100
93
64
68
57
51
44
43
47
45
49
35
27
29
36
27
32
23
33
31
39
28
29
26
30
14
18
22
21
22
24
29
29
15
26
25
24
20
20
18
18
25
16
13
22
25
19
13
26
10
15
14
15
18
10
14
16
18
15
13
10
16
15
11
17
14
18
10
11
10
11
13
11
8
16
13
21
16
8
14
10
12
12
10
11
8
13
10
10
13
14
13
7
13
12
15
13
9
10
11
11
8
12
10
14
10
17
7
9
10
13
1
7
13
14
10
15
15
7
11
11
5
9
7
9
11
11
6
10
12
10
15
7
12
4
11
4
6
7
5
8
13
10
10
8
7
13
11
5
8
12
7
9
9
11
8
2
11
9
11
8
6
14
7
10
5
9
9
11
10
7
10
8
6
11
4
7
11
13
8
8
15
8
8
10
10
3
8
9
9
6
10
11
9
13
9
12
15
14
14
11
10
13
13
6
4
10
18
10
10
8
11
7
9
11
13
13
10
10
7
17
11
13
17
15
14
19
22
16
21
22
14
19
31
16
13
10
11
8
12
18
11
8
12
12
10
12
5
14
4
6
15
14
7
14
13
13
9
14
12
16
11
16
20
12
13
12
12
21
17
13
12
14
9
11
19
12
15
15
11
11
16
14
12
18
12
21
12
18
17
21
18
27
15
23
27
29
34
30
31
27
43
43
77
105
174
266
318
504
747
1052
1326
1621
2048
2292
2419
2475
2481
2294
2020
1757
1421
1096
795
584
400
256
153
106
76
31
22
26
25
17
21
20
17
25
30
35
34
45
60
74
96
150
186
275
373
459
527
629
690
810
795
825
755
673
598
444
393
245
198
159
88
62
39
25
23
8
11
7
5
2
2
5
5
2
5
9
7
10
8
11
14
20
18
29
35
39
46
66
60
53
56
69
50
49
49
30
23
18
17
14
1
2
1
7
4
2
0
3
4
0
1
0
1
2
0
1
4
0
2
2
4
1
0
1
1
3
0
0
3
1
1
2
0
2
2
4
3
1
4
7
5
5
1
2
10
6
7
9
1
5
5
7
2
3
7
1
5
4
2
8
6
7
2
3
3
4
5
4
6
2
6
3
4
6
5
4
5
3
8
3
4
4
5
13
9
7
3
2
8
5
9
5
12
5
5
10
9
6
4
10
10
11
14
12
11
13
12
15
12
12
13
19
15
18
19
25
17
12
22
21
19
19
23
21
20
20
27
28
14
29
40
34
31
35
29
44
41
38
24
38
41
42
46
48
39
38
50
49
57
61
59
49
63
71
76
91
75
82
90
77
87
95
102
91
88
95
75
80
84
81
80
72
52
42
37
43
37
24
21
16
12
11
9
7
7
0
2
0
2
2
1
2
0
0
0
0
1
0
0
0
1
1
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
1
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
1
0
0
0
0
1
0
0
0
1
2
3
3
4
4
6
3
11
9
11
6
14
7
2
3
10
3
2
4
1
3
2
0
1
0
0
0
0
0
0
1
0
0
0
1
0
0
0
2
0
3
1
2
6
5
0
3
3
0
6
2
2
2
0
3
3
0
2
0
0
1
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
1
1
0
0
2
0
0
0
0
2
1
2
0
0
2
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
1
0
0
0
1
0
0
1
0
1
0
0
0
0
0
0
2
2
1
1
0
1
0
0
0
1
0
0
2
2
0
0
3
0
0
0
0
1
0
0
0
3
2
2
1
0
1
1
2
1
1
0
2
1
0
1
1
1
0
1
2
1
0
0
1
0
1
0
0
0
0
1
1
1
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
1
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10028.spe
Start: 21.07.2014 18:11:09
Stop: 21.07.2014 18:12:49
BESSY: 298.8045514 298.8005051
IONI: 3.493774E-11 3.379306E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 78033
Run: 100000
Dead: 22
ICR: 72300
OCR: 0
Accumulated BESSY: 59704.9140341
Accumulated Ioni: 6.85417296E-09
N Accumulation: 200
Normfactor:  3.7393544e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
 
exp_x 91.2603988647461 91.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.8901
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  28   1      7.472   4884497.      2210.     159.04
