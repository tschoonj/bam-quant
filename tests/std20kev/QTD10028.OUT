     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:42  บ   
     บ  Spectrum: QTD10028.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =   125.9   last change =   .02%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -42.3 ๑   100.0   บ     -42.3 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.262 ๑   2.000   บ    20.262 ๑    .001   บ   
     บ det NOISE (eV) บ      -73.7 ๑    40.0   บ     -73.7 ๑    -3.1   บ   
     บ  FANO factor   บ       .192 ๑    .050   บ      .192 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ni-Ka                           4884497. ๑   2210.           บ   
     บ    KA1         7.478    .66126     3229926. ๑   1462.    174.2  บ   
     บ              371.146    158.63         223.           9.47E-01  บ   
     บ    KA2         7.461    .33874     1654571. ๑    749.    159.0  บ   
     บ              370.307    158.41         224.           9.47E-01  บ   
     บ                                                                 บ   
     บ  2 Ni-Kb                            788218. ๑    888.           บ   
     บ    KB1         8.265   1.00000      788218. ๑    888.    110.2  บ   
     บ              409.987    168.48         229.           9.60E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           38617. ๑    198.           บ   
     บ    L1N2       14.944    .74128       28626. ๑    147.     26.8  บ   
     บ              739.635    236.03          91.           9.87E-01  บ   
     บ    L1O3       15.737    .23940        9245. ๑     48.      5.0  บ   
     บ              778.759    242.80          84.           9.88E-01  บ   
     บ    L1O2       16.530    .01933         746. ๑      9.       .9  บ   
     บ              817.884    249.39          69.           9.87E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
